.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,11 +1,11 @@
 blank_issues_enabled: true
 contact_links:
-  - name: Issue with the CLI installer ?
+  - name: Having problems with the CLI installer?
```

Comment:
Can you look for any reference to the "CLI Installer" or "Create Skeleton App" and modify those to say the "Skeleton CLI". This is technically a different tool, but we'll no longer be updating the former any longer. The latter replaces this.